staging/src/k8s.io/client-go/pkg/apis/clientauthentication/types.go

```diff
@@ -66,13 +66,13 @@ type ExecCredentialStatus struct {
 	ExpirationTimestamp *metav1.Time
 	// Token is a bearer token used by the client for request authentication.
 	// +optional
-	Token string
+	Token string `datapolicy:"token"`
 	// PEM-encoded client TLS certificate.
 	// +optional
 	ClientCertificateData string
 	// PEM-encoded client TLS private key.
 	// +optional
-	ClientKeyData string
+	ClientKeyData string `datapolicy:"secret-key"`
```

Comment:
is there anything that will catch a typo in these values? how does data policy handle unknown datapolicy values?

Reply:
Values of data policy are informational only. Datapolicy library will report list of all values.

Reply:
Two weeks late to the party here, but KEP-1933 is going to use these field tags for its static analysis.  It would be relatively easy to add an analysis step that examines what values correspond to `datapolicy` keys in field tags.

It would be a little messier to check for typos is the `datapolicy` key itself, but we could bark up that tree if we're concerned about it.